lib/logsource.c

```diff
@@ -395,14 +401,18 @@ log_source_queue(LogPipe *s, LogMessage *msg, const LogPathOptions *path_options
 
   log_msg_set_tag_by_id(msg, self->options->source_group_tag);
 
+
+  if (!_invoke_mangle_callbacks(s, msg, path_options))
+    return;
+
+  log_source_override_host(self, msg);
+  log_source_override_program(self, msg);
```

Comment:
I would not extract the `self->options->*_override` conditions, if the names of the extracted functions do not indicate conditionality.

For example,
```C
if (self->options->program_override)
  log_source_override_program(self, msg);
```
is less misleading in my opinion.

Reply:
Thanks.